docs/reference/ingest.asciidoc

```diff
@@ -271,6 +271,21 @@ Use the <<index-final-pipeline,`index.final_pipeline`>> index setting to set a
 final pipeline. {es} applies this pipeline after the request or default
 pipeline, even if neither is specified.
 
+[discrete]
+[[pipelines-for-beats]]
+=== Pipelines for {beats}
+
+To add an ingest pipeline to an Elastic Beat, specify the `pipeline`
+parameter under `output.elasticsearch` in `<BEAT_NAME>beat.yml`. For example,
```

Comment:
```suggestion
parameter under `output.elasticsearch` in `<BEAT_NAME>.yml`. For example,
```
```suggestion
parameter under `output.elasticsearch` in `<BEAT_NAME>beat.yml`. For example,
```

Reply:
Unless you use this somewhere else, I'd just shorten it.

Reply:
Great suggestion. Thanks @dedemorton!